test/groovy/TestNotification.groovy

```diff
@@ -11,7 +11,14 @@ class TestNotification {
     }
 
     @Test
-    void should_recognize_slack_channels() throws Exception {
+    void should_recognize_slack_channels_using_an_host_and_an_hash() throws Exception {
+        def notification = Notification.fromMaintainersFileValue("elifesciences.slack.com#xpub-tech")
```

Comment:
Could potentially make all of these URIs, so

```
mailto:p.hooper@elifesciences.org
https://elifesciences.slack.com#xpub-tech
```

Probably taking it too far though.